docker/README.md

```diff
@@ -57,27 +77,33 @@ create the API keys in the OpenQA's web interface, and store the configuration i
 
 ### Change the OpenID provider
 
-https://id.fedoraproject.org/ is set as a default OpenID provider. To change it, run:
+https://www.opensuse.org/openid/user/ is set as a default OpenID provider. To change it, run:
 
-    docker exec -ti openqa_data /scripts/set_openid
+    docker exec -it openqa_data /scripts/set_openid
```

Comment:
That is only a personal choice.
One reason is unification: in this readme file, it sometimes appeared as -it and sometimes as -ti. And I would like to make it all the same order.
The other reason is that in the docker documentation it always appear as -it. So I chose this one.

Should I revert this change?